project/AppDependencies.scala

```diff
@@ -7,7 +7,7 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     play.sbt.PlayImport.ws,
-    "uk.gov.hmrc"       %% "play-frontend-hmrc-play-30"    % "12.8.0",
+    "uk.gov.hmrc"       %% "play-frontend-hmrc-play-30"    % "12.11.0",
```

Comment:
As per platops-pr-bot comment, we can update to
"uk.gov.hmrc" %% "play-frontend-hmrc-play-30" % "12.12.0"